### PR TITLE
count_forks option for Lang cards and hide_border option for Repo cards

### PR DESCRIPTION
**Pull request by Bas950**
/api/top-langs/?username=USER
image
/api/top-langs/?username=USER&count_forks=true
image

/api/pin/?username=USER&repo=REPO
image
/api/pin/?username=USER&repo=REPO&hide_border=true
image
(Hard to see... but it is removed.)

/api/top-langs/?username=USER&count_forks=true
image
/api/top-langs/?username=USER&count_forks=true&exclude_repo=REPO
image

Added prettier configs so users don't change the whole spacings.

closes https://github.com/anuraghazra/github-readme-stats/issues/468

### CODE REVIEW

File: src/cards/repo-card.js [hunk 1]
```diff
@@ -76,78 +77,78 @@ const renderRepoCard = (repo, options = {}) => {
...
+		width: 400,
+		height,
+		colors: {
+			titleColor,
+			textColor,
+			iconColor,
+			bgColor
+		}
+	});
+
```
Comment (anuraghazra) [inline src/cards/repo-card.js:120]: Please make sure nothing else changes other than the code you want. all the code is messed up. make sure your codeeditor does not have wrong tabs/space configs

Comment (Bas950) [reply src/cards/repo-card.js:120]: > Please make sure nothing else changes other than the code you want. all the code is messed up. make sure your codeeditor does not have wrong tabs/space configs

What are the settings you use then?

**Comment by anuraghazra** [comment]
Hey @Bas950 i really would appreciate if you can add the exclude_repo feature on a different PR, let's add one feature per PR.

**Comment by Bas950** [comment]
> Hey @Bas950 i really would appreciate if you can add the exclude_repo feature on a different PR, let's add one feature per PR.

Alright, will make different PRs for each feature

**Pull request closed without merging**